nephio/optional/spire/cluster-list.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: clusters
+  namespace: spire
+data:
+  clusters.conf: |
+    clusters= {       
+      "kind" = {
+        service_account_allow_list = ["spire:spire-agent"]
+       }
+      "regional" = {
```

Comment:
why are you adding this regional cluster in the cluster list in this yaml ? This cluster list should be patched at runtime by the reconciler